kiwi/defaults.py

```diff
@@ -560,7 +560,7 @@ def get_grub_path(root_path, filename, raise_on_error=True):
             for install_dir in install_dirs:
                 grub_path = os.sep.join(
                     [root_path, install_dir, grub_name, filename]
-                )
+                ).replace('//', '/')
```

Comment:
Just a matter of taste, but isn't more generic and easier to just use `os.path.normpath` instead of this replacement?

Reply:
I wanted to do that, but os.path.normpath does not delete double slashes at the beginning :( I think this is a bug there (or intentional). It works fine if the double slashes are in between path elements but not at the beginning. You can give it a try with a simple script. I did the same and was astonished. That's why it ended up with this stupid line of code

Reply:
I'm happy if you know a nicer solution. I found the file.Path class which works nicely but this would create another dependency which I don't want

Reply:
but maybe the above can be changed to use os.path() I'll give it a try

Reply:
@davidcassany  ok better solution, pushed. How about it now ?